### PR TITLE
Add support for Action Workflow state updates

### LINKED ISSUE #1637: Add Workflow enable and disable states
Github has two APIs to manually enable and disable a workflow by ID or Filename,

- [Enable a workflow](https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#enable-a-workflow)
- [Disable a workflow](https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#disable-a-workflow)

Both are not implemented yet. I would like to take this up if it is supposed to be added.

### PR DESCRIPTION
**Pull request by n1lesh**
Fixes #1637  

Includes enable/disable by ID and Filename w/ tests.

### CODE REVIEW

File: github/actions_workflows.go
Comment (gmlewis) [inline github/actions_workflows.go:176]:
```suggestion
// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#enable-a-workflow
```

File: github/actions_workflows.go
Comment (gmlewis) [inline github/actions_workflows.go:193]:
```suggestion
// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#disable-a-workflow
```

File: github/actions_workflows.go
Comment (gmlewis) [inline github/actions_workflows.go:201]:
```suggestion
// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#disable-a-workflow
```

File: github/actions_workflows.go
Comment (gmlewis) [inline github/actions_workflows.go:191]:
```suggestion
// DisableWorkflowByID disables a workflow and sets the state of the workflow to "disabled_manually".
```

File: github/actions_workflows.go
Comment (gmlewis) [inline github/actions_workflows.go:199]:
```suggestion
// DisableWorkflowByFileName disables a workflow and sets the state of the workflow to "disabled_manually".
```

File: github/actions_workflows.go
Comment (gmlewis) [inline github/actions_workflows.go:202]:
```suggestion
func (s *ActionsService) DisableWorkflowByFileName(ctx context.Context, owner, repo, workflowFileName string) (*Response, error) {
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
This looks great, @n1lesh - just a few minor tweaks, please... mainly to help keep the auto-generated godocs to read nicely and be consistent with the rest of the repo.

**Review by gmlewis — CHANGES_REQUESTED** [review]
Please revert the addition of ".DS_Store" to this PR.
In fact, if you want to add ".DS_Store" to the ".gitignore" file in this PR instead, that would prevent this from happening again.  Thanks.

**Comment by n1lesh** [comment]
> Please revert the addition of ".DS_Store" to this PR.
> In fact, if you want to add ".DS_Store" to the ".gitignore" file in this PR instead, that would prevent this from happening again. Thanks.

Should have noticed this before committing. My apologies!

**Review by gmlewis — APPROVED** [review]
> Should have noticed this before committing. My apologies!

No problem!  Thank you, @n1lesh !
LGTM.

Awaiting second LGTM before merging.

**Review by nightlark — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @nightlark !
Merging.

**Pull request merged**
